src/photos/ducks/timeline-clusters/index.js

```diff
@@ -18,6 +19,7 @@ const TIMELINE_QUERY = client =>
     .sortBy({
       'metadata.datetime': 'desc'
     })
+    .include(['albums'])
```

Comment:
If I remember correctly, we have a bug in `CozyClient` with the `include` keywords :  CozyClient generates too much requests instead of doing its optimizations to make only one or two requests. 

Is it something you saw?